### PR TITLE
[4.0] com_banners - remove unused submenu

### PR DESCRIPTION
**Pull request by travisrisner**
Remove unused references to `addSubmenu`in various views on com_banners

### CODE REVIEW

File: administrator/components/com_banners/View/Banners/HtmlView.php [hunk 1]
```diff
@@ -81,8 +81,6 @@ public function display($tpl = null)
 			throw new GenericDataException(implode("\n", $errors), 500);
 		}
 
-		BannersHelper::addSubmenu('banners');
```
Comment (Quy) [inline administrator/components/com_banners/View/Banners/HtmlView.php:84]: Remove `use Joomla\Component\Banners\Administrator\Helper\BannersHelper;`. Repeat in the other files.

Comment (travisrisner) [reply administrator/components/com_banners/View/Banners/HtmlView.php:84]: Thanks @Quy

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 4e5c00deeee4a5f1b7bdbc6e93511a454d40097b<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25266">issues.joomla.org/tracker/joomla-cms/25266</a>.</sub>

**Comment by mothershedc** [comment]
I have tested this item :white_check_mark: successfully on [`4e5c00d`](https://github.com/joomla/joomla-cms/commit/4e5c00deeee4a5f1b7bdbc6e93511a454d40097b)

**Comment by ghost** [comment]
Status "Ready To Commit".

**Comment by roland-d** [comment]
Thank you.

**Pull request merged**